### PR TITLE
Make /docs consistent with docstring edits

### PR DESCRIPTION
**Pull request by lebovic**
Updates `/docs` files to be consistent with edits made to docstrings in PR #2

### CODE REVIEW

**Review by bcai2 — CHANGES_REQUESTED** [review]
LGTM; there should also be a couple edits to make in `/docs/user/quickstart.rst`, but they should look the same as these.

File: docs/user/quickstart.rst [hunk 1]
```diff
@@ -43,9 +43,9 @@ command line:
 Once Toolchest is configured, you can do the same with the corresponding
 Toolchest function:
 
->>> import toolchest_client as tc
->>> tc.your_tool(
-...     YOUR_CUSTOM_TOOL_ARGS,
+>>> import toolchest_client as toolchest
+>>> toolchest.your_tool(
+...     tool_args="YOUR_CUSTOM_TOOL_ARGS",
```
Comment (lebovic) [inline docs/user/quickstart.rst:48]: @bcai2 I think those edits are in this PR already!

Comment (bcai2) [reply docs/user/quickstart.rst:48]: Oops, gotcha

**Review by bcai2 — APPROVED** [review]
LGTM

**Pull request merged**